openbas-front/src/admin/components/profile/UserForm.js

```diff
@@ -96,6 +96,9 @@ const UserForm = ({ onSubmit, initialValues }) => {
             <MenuItem key="fr" value="fr">
               {t('French')}
             </MenuItem>
+            <MenuItem key="zh" value="zh">
+              {t('Chinese')}
```

Comment:
The translation is added on the profile but shouldn't we add it to the list of default languages of the platform too ?

Reply:
I am not sure to understand your point
It's already available in the AppIntlProvider class

Reply:
Oh, I missed the BrownserLanguage and the moment configuration